spec/fog/compute/xen_server/models/server_spec.rb

```diff
@@ -508,7 +508,7 @@ def server.wait_for(&block); instance_eval(&block); @halted = true end
     describe "when it can be cloned" do
       before :each do
         def server.can_be_cloned?; true end
-        def service.clone_vm(name, reference); @cloned = true end
+        def service.clone_vm(reference, name); @cloned = true end
```

Comment:
Unused method argument - `reference`. If it's necessary, use `_` or `_reference` as an argument name to indicate that it won't be used. You can also write as `clone_vm(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used. You can also write as `clone_vm(*)` if you want the method to accept any arguments but don't care about them.